src/main/java/roomescape/controller/api/ReservationController.java

```diff
@@ -0,0 +1,70 @@
+package roomescape.controller.api;
+
+import java.util.List;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+import roomescape.dto.ReservationRequest;
+import roomescape.dto.ReservationResponse;
+import roomescape.exception.CannotCreatedException;
+import roomescape.exception.DuplicateReservationException;
+import roomescape.exception.NotCorrectDateTimeException;
+import roomescape.exception.ThemeDoesNotExistException;
+import roomescape.exception.TimeDoesNotExistException;
+import roomescape.service.ReservationService;
+
+@RestController
+@RequestMapping("/reservations")
+public class ReservationController {
+
+    private final ReservationService reservationService;
+
+    private ReservationController(ReservationService reservationService) {
+        this.reservationService = reservationService;
+    }
+
+    @PostMapping
+    @ResponseStatus(HttpStatus.CREATED)
+    public ReservationResponse createReservation(@RequestBody ReservationRequest request) {
+        return ReservationResponse.from(reservationService.addReservationAfterNow(request));
+    }
+
+    @PostMapping("/admin")
+    @ResponseStatus(HttpStatus.CREATED)
+    public ReservationResponse createAdminReservation(@RequestBody ReservationRequest request) {
+        return ReservationResponse.from(reservationService.addReservation(request));
+    }
+
+    @GetMapping
+    public List<ReservationResponse> readReservations() {
+        return reservationService.findAllReservations().stream()
+            .map(ReservationResponse::from)
+            .toList();
+    }
+
+    @DeleteMapping("/{id}")
+    @ResponseStatus(HttpStatus.NO_CONTENT)
+    public void deleteReservation(@PathVariable Long id) {
+        reservationService.removeReservation(id);
+    }
+
+    @ExceptionHandler(value = {TimeDoesNotExistException.class, ThemeDoesNotExistException.class,
+        NotCorrectDateTimeException.class})
```

Comment:
```suggestion
    @ExceptionHandler(value = {
        TimeDoesNotExistException.class,
        ThemeDoesNotExistException.class,
        NotCorrectDateTimeException.class})
```
줄바꿈을 잘 활용해보면 읽기 더 쉬운 코드가 될 것 같습니다.
소소해 보여도 코드가 많아지고 복잡해질수록 중요하더라고요 :)

Reply:
전달하고자 하는 인자 줄바꿈에도 신경써야겠군요! 
리팩터링 하면서 이 부분의 ExceptionHandler는 제거하게 되어서 `GlobalExceptionHandler`에 적용해보았습니다. 👍🏻